bio/bedtools/complement/meta.yaml

```
@@ -0,0 +1,12 @@
+name: bedtools complement
+description: >
+  ``Bedtools complement`` maps all regions of the genome which are not covered by the input.
+authors:
+  - Antonie Vietor
+input:
+  - BED/GFF/VCF files
+  - genome file (`genome file format <https://bedtools.readthedocs.io/en/latest/content/general-usage.html#genome-file-format>`_)
+output:
+  - .bed.complement or
+  - .vcf.complement or
+  - .gff.complement
```

I would always keep the file format extension last (see suggestion below in the Snakefile) and here I would simply keep the description as generic as for the input:
```suggestion
  - complemented BED/GFF/VCF file
```
Similarly in the other wrappers in these PRs.